package-lock.json

```diff
@@ -294,7 +294,6 @@
       "version": "2.2.0",
```

Comment:
Don't push both lock file

and please program to force devs to use yarn or npm only one the other

https://qiita.com/suin/items/a7bf214f48eb9b2d9afc